python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -696,15 +524,16 @@ def cloneVal(v):
         if isinstance(new_val, Node):
             new_val._setParent(obj)
 
-        obj.setNodeAttribute(_getName(self), new_val)
+        obj.setSchemaAttribute(_getName(self), new_val)
 
         obj._setDirty()
 
     def __delete__(self, obj):
         """
         Delete an attribute from teh Descriptor(?) and Node
         """
-        obj.removeNodeAttribute(_getName(self))
+        raise Exception('Descriptor.__delete__')
+        del obj._data[_getName(self)]
```

Comment:
Ah, bugger. I put that exception in as a local test to see if `__delete__` is ever even called as I don't think that it should be. I didn't mean to commit it. I will remove that line but in the future I'd like to remove the method entirely.